### DISCUSSION TITLE
Hackeada

### DISCUSSION THREAD

**Discussion by Selfine722** [community/community#146796] [Code Security]
Quisiera saber si tengo hameado mi teléfono?

**Comment by ghostinhershell** [comment]
Hi @Selfine722,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on issues like this. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space is not the right place for this particular topic.

Best of luck!
